src/objects.rs

```diff
@@ -1809,6 +1812,26 @@ impl AddPrefix for GridRelCalendarLine {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, Copy, PartialEq)]
+pub struct TzExt(pub Tz);
```

Comment:
It's worth commenting why you had to do that, and maybe rename:
```suggestion
pub struct ParsableTz(pub Tz);
```

Reply:
we may have other needs so I prefer a generic name